e2e/volume_ops_test.go

```diff
@@ -405,6 +410,12 @@ func TestVolumeOptions(t *testing.T) {
 	settableKey := "afr.use-compound-fops"
 	optionReq.Options = map[string]string{settableKey: "on"}
 	r.Nil(client.VolumeSet(volname, optionReq))
+
+	var resetOptionReq api.VolOptionResetReq
+	resetOptionReq.Options = []string{"afr.use-compound-fops"}
+	resetOptionReq.Force = true
+	r.Nil(client.VolumeReset(volname, resetOptionReq))
+
```

Comment:
Please add a test case that exercises the `All` option

Reply:
Done.